src/components/Panel.scss

```diff
@@ -228,7 +229,12 @@ $Panel-header-text-color: $cu-info !default;
 
         .Panel-cell--header {
           border-right: 0; // 3
-          border-bottom: 1px solid $cu-divider--light;  // 3
+          padding-bottom: 0;
+          background-color: inherit;
+        }
+
+        .Panel-cell--header + .Panel-cell {
+          padding-top: $su-small;
         }
```

Comment:
We simplified the design of this modifier. It was designed for use in two-column tables, and is currently being battle tested in the account section. The cell headers background color and borders were feeling too busy, so we made the cell headers act more like simple text labels.

![stacked-account](https://user-images.githubusercontent.com/9888467/31139566-942a00e6-a837-11e7-92f0-46fdecae638b.gif)


![stacked](https://user-images.githubusercontent.com/9888467/31139405-0ba2fbf6-a837-11e7-9bfe-c0fdd70c7ce6.gif)